distribution/ddf-common/src/main/resources/etc/artemis-backup.xml

```diff
@@ -13,13 +13,21 @@
                xmlns="urn:activemq"
                xsi:schemaLocation="urn:activemq /schema/artemis-configuration.xsd">
 
-    <jms xmlns="urn:activemq:jms">
-        <queue name="DLQ"/>
-        <queue name="ExpiryQueue"/>
-    </jms>
-
     <core xmlns="urn:activemq:core">
 
+        <addresses>
+            <address name="DLQ">
+                <anycast>
```

Comment:
Artemis changed how their messaging model works. https://activemq.apache.org/artemis/docs/2.1.0/address-model.html

When an address has the "anycast" routing type, its queues will receive messages in a point to point manner, similar to how a normal would queue work.
When an address has the "multicast" routing type, its queues will receive messages in a publish/subscribe manner, similar to how a normal would topic work.